homeassistant/components/reolink/__init__.py

```diff
@@ -79,6 +79,9 @@ async def async_device_config_update():
 
     async def async_check_firmware_update():
         """Check for firmware updates."""
+        if not host.api.supported(None, "update"):
+            return False
```

Comment:
Does this return value match the return statement below? It would be nice to add type annotations to this function.